otcextensions/common/utils.py

```diff
@@ -21,12 +21,11 @@
 # import json
 import os
 # import re
-# import six.moves.urllib.parse as urlparse
+# import urllib.parse as urlparse
 # import sys
```

Comment:
do we need  to store that commented items?

Reply:
nope, we do not need them. I forgot to comment on that also.

Reply:
fixed